docs/UserGuide.md

```diff
@@ -206,9 +181,30 @@ Expected outcome:
 
 <img src="images/checkmc.png" width="550px" height="350px">
 
+### View All Required Modules : `required`
+
+Display all required modules in the Computer Science syllabus onto the `Command Line Display` for easy referencing.
+
+Format: `required`
+
+Example of usage: `required`
+
+Expected outcome:
+
+### Search for a Required Module : `search`
+
+Search for a required module in the Computer Science syllabus using the module code and display the module details 
+(module code, title, modular credit, description, preclusion, prerequisite) in the `Command Line Display`.
```

Comment:
Perhaps it might be better to list this out separately? Maybe below this sentence we can put:

The following module details will be displayed:
* module code
* title
* etc